scripts/check_and_deploy_kubectl.sh

```diff
@@ -107,18 +108,29 @@ echo -e "Namespace ${CLUSTER_NAMESPACE} authorizing with private image registry
 #Update deployment.yml with image name
 echo "=========================================================="
 echo "CHECKING DEPLOYMENT.YML manifest"
-if [ -z "${DEPLOYMENT_FILE}" ]; then DEPLOYMENT_FILE=deployment.yml ; fi
-if [ ! -f ${DEPLOYMENT_FILE} ]; then
-    echo -e "${red}Kubernetes deployment file '${DEPLOYMENT_FILE}' not found${no_color}"
+if [ -z "${DEPLOYMENT_FILE}" ]; then 
+    DEPLOYMENT_FILE=deployment.yml ; 
+fi
+echo "CHECKING DEPLOYMENT_SUBDIRECTORY"
+if [ -z "${DEPLOYMENT_SUBDIRECTORY}" ]; then
+    DEPLOYMENT_SUBDIRECTORY=./
+else
+    if [[ $DEPLOYMENT_SUBDIRECTORY =~ //$/ ]]; then
+    else
```

Comment:
There is a failure using pipeline build image as there is no then commands to execute